src/components/Auth/Forms/register.tsx

```diff
@@ -174,7 +174,7 @@ const RegisterForm = () => {
 
             <div className="m-1 text-center">
               <Link href="/auth/login">
-                <span className="text-xs text-gray-400 underline">
+                <span className="text-[13px] text-gray-400 underline">
```

Comment:
We recommend not using arbitrary values but if these are due to spacing issues/line-height issues, check the leading classes